### DISCUSSION TITLE
Hej

### DISCUSSION THREAD

**Discussion by jalabuk** [Part-DB/Part-DB-server#224] [Q&A] [answered]
Hej i installed this on my web host works whit login but don't se nothing it's like acces denied 

https://www.lagertek.shop/public

How can I fix this ? If somebody could help out 

Thx

**Comment by jbtronics** [comment, marked as answer]
It is recommended that you put Part-DB in the root of its own (subdomain) like partdb.webpage.de, and that's what's the default configuration assumes.

If you really need to put Part-DB into a (virtual) webfolder, than you have to configure the |.setPublicPath` and `.setManifestKey` option in the `webpack.config.js` line 48 and 49. There is some comment in the file on what to do exactly.

Afterwards you have to recompile your frontend assets with `yarn build`.
Otherwise your browser will not find Part-DB Stylesheets and JavaScript and you will get the unstyled pages you are currently seeing.

**Reply by jalabuk** [reply]
A okay nice now it's working thx for helping :) 

http://public.lagertek.shop

How can I make it go direct in the public folder when just typing www.lagertek.shop 
I don't have access too the apache configuration on this web host any other ways too du it ?

**Reply by jbtronics** [reply]
You can just put a file `index.html` on your www.lagertek.shop webserver and put a HTML redirect directive into it. See here https://www.w3docs.com/snippets/html/how-to-redirect-a-web-page-in-html.html

**Reply by jalabuk** [reply]
Okay thx a lot :)
